sdcm/nemesis.py

```diff
@@ -1000,9 +1001,32 @@ def disrupt_restart_with_resharding(self):
         self.log.info('Set back murmur3_partitioner_ignore_msb_bits value to 12')
         self.target_node.restart_node_with_resharding()
 
+    def replace_full_file_name_to_prefix(self, one_file, ks_cf_for_destroy):
+        # The file name like: /var/lib/scylla/data/scylla_bench/test-f60e4f30c98f11e98d46000000000002/mc-220-big-Data.db
+        # or /var/lib/scylla/data/scylla_bench/test-f60e4f30c98f11e98d46000000000002/mc-3g6x_0sic_4r1eo23d0mkrb3fs2l-big-Data.db
+        # For corruption we need to remove all files that their names are started from "mc-220-" or "mc-3g6x_0sic_4r1eo23d0mkrb3fs2l-"
+        # (MC format)
+        # Old format: "system-truncated-ka-" (system-truncated-ka-7-Data.db)
+        # Search for these prefixes
+        file_name = os.path.basename(one_file)
+
+        try:
+            file_name_template = re.search(r"([^-]+-[^-]+)-", file_name).group(1)
+        except Exception as error:  # pylint: disable=broad-except
+            self.log.debug('File name "{file_name}" is not as expected for Scylla data files. '
+                           'Search files for "{ks_cf_for_destroy}" table'.format(file_name=file_name,
+                                                                                 ks_cf_for_destroy=ks_cf_for_destroy))
+            self.log.debug('Error: {}'.format(error))
+            return ""
+
+        file_for_destroy = one_file.replace(file_name, file_name_template + '-*')
+        self.log.debug('Selected files for destroy: {}'.format(file_for_destroy))
+        return file_for_destroy
+
```

Comment:
This part that handle the file name should be split into a separate commit.
We need this for all "DestroyData" nemesis.

Reply:
(and also the parts below) but without the new nemesis.

Reply:
If so it's better to create a separate PR

Reply:
let's do it in a separate PR

Reply:
done